SPECS-EXTENDED/gspell/gspell.spec

```diff
@@ -4,16 +4,17 @@ Distribution:   Azure Linux
 %global gtk3_version 3.20
 
 Name:           gspell
-Version:        1.8.4
-Release:        2%{?dist}
+Version:        1.14.0
+Release:        1%{?dist}
 Summary:        Spell-checking library for GTK+
 
-License:        LGPLv2+
-URL:            https://wiki.gnome.org/Projects/gspell
-Source0:        https://download.gnome.org/sources/%{name}/1.8/%{name}-%{version}.tar.xz
+License:        LGPLv2.1+
+URL:            https://gitlab.gnome.org/GNOME/gspell
```

Comment:
Please try to use `gnome.org` for fetching source for security purposes.

Reply:
I was simply following the link that was provided on the gnome.org website:
![image](https://github.com/user-attachments/assets/92c9e4c1-aea5-4d0d-8d79-7af9a3e0e4bc)
I am happy to change it back if that is preferred, though.